src/modules/tools/endstops/Endstops.cpp

```diff
@@ -186,14 +197,16 @@ bool Endstops::load_old_config()
             // add index to the homing struct if this is the one used for homing
             if((hinfo.home_direction && j == MIN_PIN) || (!hinfo.home_direction && j == MAX_PIN)) hinfo.pin_info= info;
 
+            info->is_max_stop = j == MAX_PIN;
```

Comment:
Whether a limit is min or max has nothing to do with the homing direction, you need to totally disconnect this concept of homing direction with limit switches

Reply:
I guess this is the only way to do it with the old syntax